lib/gems/pending/util/object_storage/miq_s3_storage.rb

```diff
@@ -84,8 +84,8 @@ def bucket
   private
 
   def s3
-    require 'aws-sdk'
-    require 'util/extensions/aws-sdk/s3_upload_stream_patch'
+    require 'aws-sdk-s3'
```

Comment:
Do you want to use `require "aws-sdk-#{service.to_s.downcase}"` here as well? Or is that overkill?

Reply:
where I can take `service` here? :)

Reply:
True enough!